Resources/views/submission/overview.html.twig

```diff
@@ -33,21 +33,25 @@
         <thead>
             <tr>
                 <th>Id</th>
+                <th>Label</th>
                 <th>Form</th>
                 <th>Instance</th>
                 <th>Locale</th>
                 <th>Created</th>
+                <th>Expiration</th>
```

Comment:
I would call this deadline ? Expiration has too many other interpretations.

Reply:
OK I changed it